app/Repository/Member/Member.php

```diff
@@ -246,13 +239,25 @@ public function __get( $name ) {
 			return $this->groups;
 		}
 		
-		if ( 'rootGroup' === $name ) {
-			return $this->rootGroup;
+		if ( 'rootGroups' === $name ) {
+			return $this->getRootGroups();
 		}
 		
 		return $this->getField( $name );
 	}
 	
+	/**
+	 * Returns an array with the root groups of this member.
+	 *
+	 * Access this function using magic property access.
+	 *
+	 * @return null|Group[]
```

Comment:
Wann kommt null zurück? Wenn der User keine rootGroup hat (da er in keiner Gruppe ist, falls das überhaupt möglich ist), würde ich eher ein leeres Array erwarten.
Und bei einem Fehler gibt es ja eh eine Exception

Reply:
Du hast recht, ein leerer Array ergibt mehr Sinn als null. Ich ändere das gleich.